examples/chroot_vm.c

```diff
@@ -17,6 +17,8 @@
 #define MAX_PATH 4096
 #endif
 
+#define NET_MODE KRUN_NET_MODE_TSI
+
```

Comment:
Instead of a build time conditional, please turn this into a command line flag. This might require finally adopting something like `getopt()` to process the command line, but I think it's worth it.